docs/references/ingress-messages.mdx

```diff
@@ -10,25 +10,27 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
 ## Overview
 
-Ingress messages are calls to ICP canisters that are sent by external entities, such as users, usually through an agent like [agent-js](https://github.com/dfinity/agent-js/tree/main) that runs within a web application, or [agent-rs](https://github.com/dfinity/agent-rs/tree/main) in a Rust application. See also the introductory [call overview](/docs/current/developer-docs/smart-contracts/call/overview). 
+Ingress messages are calls to ICP canisters that are sent by external entities, such as users, usually through an [agent](/docs/current/developer-docs/developer-tools/dev-tools-overview/#agents) like [agent-js](https://github.com/dfinity/agent-js/tree/main) that runs within a web application, or [agent-rs](https://github.com/dfinity/agent-rs/tree/main) in a Rust application. See also the introductory [call overview](/docs/current/developer-docs/smart-contracts/call/overview). 
```

Comment:
Are ingress messages limited to calls everywhere in our docs or are queries also ingress messages in some places? For sure our code is a bit ambiguous, e.g., https://github.com/dfinity/ic/blob/master/rs/validator/ingress_message/src/lib.rs

(if we're sure we use ingress message for calls only in our docs then that's great, I'm not implying we should rename code or the docs, I simply don't know if we're using it consistently)

Reply:
I see that we later use it for queries as well... So maybe should use "request" here instead of "call"?